platform/darwin/src/MGLLoggingConfiguration_Private.h

```diff
@@ -11,6 +15,13 @@
         [[MGLLoggingConfiguration sharedConfiguration] logCallingFunction:function functionLine:line messageType:type format:(message), ##__VA_ARGS__]; \
     } \
 }
+#define MGLAssert(expression, message, ...) \
+    __extension__({ \
+        if (__builtin_expect(!(expression), 0)) { \
+            MGLLogFault(message, ##__VA_ARGS__); \
+        } \
+        NSAssert(expression, message, ##__VA_ARGS__); \
```

Comment:
What about common variants like `NSCAssert()`?

Reply:
I think we can create our own `MGLCAssert`.